ubuntu/start.sh

```diff
@@ -54,11 +54,14 @@ trap 'cleanup; exit 130' INT
 trap 'cleanup; exit 143' TERM
 
 echo Determining matching VSTS agent...
+set +e
 VSTS_AGENT_URL=$(curl -LsS \
   -u user:$(cat "$VSTS_TOKEN_FILE") \
   -H 'Accept:application/json;api-version=3.0-preview' \
   "https://$VSTS_ACCOUNT.visualstudio.com/_apis/distributedtask/packages/agent?platform=linux-x64" \
-  | jq -r '.value | map([.version.major,.version.minor,.version.patch,.downloadUrl]) | sort | .[length-1] | .[3]')
+  | jq -rRs '.value | map([.version.major,.version.minor,.version.patch,.downloadUrl]) | sort | .[length-1] | .[3]' 2>/dev/null)
+set -e
+
 if [ -z "$VSTS_AGENT_URL" -o "$VSTS_AGENT_URL" == "null" ]; then
   echo 1>&2 error: could not determine a matching VSTS agent
```

Comment:
I had some additional information in this error message that I think we'll want to add to resolve #70.

Reply:
Oh, and the extra args to jq are good - I had hit that error too and wasn't sure how to resolve it.

Reply:
Ah, correct. I somehow forgot to add the actual error message back as I moved towards your suggestion. Should be fine now. 

PS- Im still working on getting access to this repo on Dockerhub so I can contribute the compute time to actually build these